agents/node/vcxagent-core/test/out-of-band.spec.js

```diff
@@ -33,4 +36,48 @@ describe('test out of band communication', () => {
     expect(payloadAlice['@type']).toBeDefined()
     expect(payloadAlice.content).toBe('Hello Alice')
   })
+
+  it('Faber issues credential via OOB', async () => {
+    try {
+      const {alice, faber} = await createAliceAndFaber()
+      await faber.createCredDef(undefined, undefined)
+      console.log(`Creating oob offer`)
+      const oobCredOfferMsg = await faber.createOobCredOffer()
+
+      await connectViaOobMessage(alice, faber, oobCredOfferMsg)
+
+      console.log(`Going to accept credential offer`)
```

Comment:
Probably forgot to remove debugging messages

Reply:
removed